src/System.Memory/tests/Memory/CustomMemoryForTest.cs

```diff
@@ -31,7 +31,7 @@ public CustomMemoryForTest(T[] array, int offset, int length)
 
         public int OnNoRefencesCalledCount => _noReferencesCalledCount;
 
-        public override int Length => _length;
+        public override Memory<byte> Memory => CreateMemory(_length);
```

Comment:
Is this necessary?
`public virtual Memory<T> Memory => CreateMemory(this, GetSpan().Length);` should still work, no?

Reply:
Done